modules/op-additional-options-webhooks.adoc

```diff
@@ -0,0 +1,150 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-additional-options-webhooks_{context}"]
+= Setting additional options for webhooks
+
+Optionally, you can set several additional options for the webhooks that several controllers in {pipelines-sshortname} create. The available options are `failurePolicy`, `timeoutSeconds`, and `sideEffects`. For information about these options, see the https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/[Kubernetes documentation].
```

Comment:
We use the word "Option" a lot here. Could we maybe revise? Also, note that the attribute {pipelines-sshortname} does not render, likely due to the additional "s". Also note use of "link:" before the link (unless pipelines docs do something different here).

This is just me playing around with wording. Maybe you want to fix it or maybe not.

```suggestion
Optionally, you can set the `failurePolicy`, `timeoutSeconds`, or `sideEffects` options for the webhooks created by several controllers in {pipelines-shortname}. For more information about these options, see the link:https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/[Kubernetes documentation].
```